scripts/wandb_logger.py

```diff
@@ -0,0 +1,28 @@
+import wandb
+import numpy as np
+from typing import Any, Dict, Tuple, Union
+from stable_baselines3.common.logger import KVWriter
+
+
+class WandBOutputFormat(KVWriter):
+    """
+    Dumps key/value pairs onto WandB.
+    """
+
+    def write(
```

Comment:
Where is this method used? I am testing the code and wandb only monitors me the resources used by the PC and the terminal output, but I can't get the graphs of the rest of the experiment. What am I doing wrong?

Reply:
This is used on line [356](https://github.com/kad99kev/sinergym/blob/8bf397163eb0e54eb1c51b49b0d7177d3cfa4241/scripts/DRL_wandb.py#L356). It has to be added to the model using the `set_logger` function.

Reply:
I have tried with PPO, in this way:

`python DRL_wandb.py -env Eplus-5Zone-hot-continuous-v1 -ep 2 -alg PPO --logger --log_interval 200 -pr ugr-sail -e alex_ugr`

Reply:
Could you try `--log_interval 1` instead of 200? I was running some experiments yesterday, and I noticed when I set it to 100, it did not log. When you set it to 1, it will log after every 2048 steps. Can you try this and let me know?

Reply:
That's right! By setting the `log_interval` to 1, I have logged the values. Thank you very much! :)

Reply:
I only have one point in SAC grahps. I think the problem is not with the WandB module, but with the callback it uses to make the write calls (`LoggerCallback`), since it was originally intended for Tensorboard. I'll have to take a look, but the script you are doing in this PR seems to be working correctly.

Reply:
Okay, thank you! I will make the changes you suggested on the other parts of the script in the meantime. I should have the PR ready by new year's.